src/katgpucbf/xbgpu/bsend.py

```diff
@@ -194,30 +197,41 @@ def _inc_counters(
             format of (n_channels_per_substream, samples_per_spectra, COMPLEX).
         data_dtype
             The `np.dtype` of the beam data transmitted.
+        enabled
+            Boolean flag array indicating which streams are enabled for transmission.
         output_names
-            List of beam stream names that are enabled for transmission for
-            this `future`.
+            List of beam stream names
         saturated
             Saturation count for the chunk for each stream in `output_names`.
+        sensors
+            Server's katcp sensors
+        sensor_timestamp
+            Timestamp (UNIX time) to use for sensor update
```

Comment:
Full stops at the end of these descriptions, please (at least to fit with the rest of them).